mozilla_version/gecko.py

```diff
@@ -343,6 +359,20 @@ class FennecVersion(_VersionWithEdgeCases):
         'build_number': 1,
     })
 
+    def __attrs_post_init__(self):
+        """Ensure attributes are sane all together."""
+        # Versions matching 68.XbN are expected since bug 1523402.
+        if (
+            self.major_number == 68 and self.is_beta and self.patch_number is None and
+            self.minor_number > 0
+        ):
+            return
+
+        if self.major_number >= 69:
+            raise PatternNotMatchedError(self, pattern='Last Fennec version is 68')
```

Comment:
This one could be worrisome if we use this for GeckoView anywhere, but I'm not going to block on it since the error message should be clear what is wrong.

Reply:
Good call. Geckoview should use MAJOR.MINOR.BUILD_ID which is parsed by https://github.com/mozilla-releng/mozilla-version/blob/e5400f31f7001bd48fb6e17626905147dd4c17d7/mozilla_version/maven.py#L13. I searched mozilla-central, but the python doesn't compute anything before https://searchfox.org/mozilla-central/rev/7e158713cf5a8514fa8161dd4a239737b05da64d/mobile/android/geckoview/build.gradle#52 happens.

In short, I don't expect any breakage, if there's one, we should probably address it by expanding MavenVersion.